core/src/main/scala/org/apache/spark/util/AkkaUtils.scala

```diff
@@ -147,7 +146,7 @@ private[spark] object AkkaUtils extends Logging {
   def askWithReply[T](
       message: Any,
       actor: ActorRef,
-      timeout: FiniteDuration): T = {
+      timeout: RpcTimeout): T = {
```

Comment:
I suggest reverting the changes to `AkkaUtils`. These methods you changed won't be called any more.